pointmatcher/ErrorMinimizers/PointToGaussian.cpp

```diff
@@ -56,9 +56,29 @@ typedef Parametrizable::ParametersDoc ParametersDoc;
 template<typename T>
 PointToGaussianErrorMinimizer<T>::PointToGaussianErrorMinimizer(const Parameters& params):
 	PointToPlaneErrorMinimizer<T>(PointToGaussianErrorMinimizer::availableParameters(), params),
-	confidenceInPenalties(Parametrizable::get<T>("confidenceInPenalties")),
-	force2D(Parametrizable::get<bool>("force2D"))
+	//confidenceInPenalties(Parametrizable::get<T>("confidenceInPenalties")),
```

Comment:
Why is this line commented? Are these the penalties described in your Subarctic forests mapping paper?

Reply:
This will be fixed during the summer I hope. The point-to-gaussian minimizer was sleeping during our work on SubT, but it is being worked on right now. There was an issue with numerical stability causing the whole thing to diverge (somehow the computers still don't like dividing by zero, dunno what's the problem...)